index.html

```diff
@@ -6,32 +6,48 @@
     <title>Cocktail Builder</title>
```

Comment:
On load of the page, if I have ingredients in my bar, I don't see any cocktails that I can mix. That's a bug.